Oasis: Rework the TEG

F1restar4 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Reworked the TEG layout and piping. Removed the old burn chamber entirely and decorated the small nook created by it's absence

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Massively simplify the TEG on Oasis to help with making atmospherics feel more approachable to new players who may be severely intimidated by the form it has taken here. The old burn chamber is not needed because of the horizontal burn chamber and is also capable of TEGloosing.

## Technical details
<!-- Summary of code changes for easier review. -->
YAML

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="1363" height="1166" alt="image" src="https://github.com/user-attachments/assets/2e0d4a96-7843-463d-8686-9b8b2c9ecd14" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->
None

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
MAPS:
- tweak: On Oasis, reworked the TEG

ArtisticRoomba commented:
> The old burn chamber is not needed because of the horizontal burn chamber and is also capable of easily TEGloosing.

generally speaking mappers put intentional friction points in maps - excessively QOLing something out of existence isn't fun and is missing the entire point of SS14. not to say that the TEG meme wasn't bad for player learning, it was, but it was funny.

i'm fine with killing the meme (if @IProduceWidgets agrees that he's had his fun), but whatever replaces it should be up to standards

> Massively simplify the TEG on Oasis to help with making atmospherics feel more approachable to new players

while this may be your intent you have actually made it just as bad as the obfuscated variant:
- no default mixing setup for canmixing has been provided
- only one can of gas, plasma, no other gasses like CO2 or oxygen are provided
- no default setups for cold/hot loop
- no overpressure venting for hot/cold loop as seen on some stations
- burn chamber lacks standard air injector fuel line input
- no welding tank nearby so atmosians enjoy a little sisyphean struggle (this was a problem with the previous iteration but im still pointing it out)

all-in-all the roundstart setup should be pretty easy to roll with. i recommend looking at amber station for a good example of the various infrastructure the TEG should have.

F1restar4 commented:
>     * no default setups for cold/hot loop
I think that default pipe setups are also a form of excessive QOL, the requirement to actually construct the loops manually adds more to the roundstart TEG setup rather than only needing to fill the loops and light the burn chamber. I don't think it should be common place to remove these aspects (What if the AME was pre-setup on every station or chem started with all precursor chems available in their locker). People tend learn better when they are hands-on building it rather than just being told what does what. Pre built TEGs remove work from atmos techs who are already lacking on stuff to do half the time.

### Changes
- Added a welding fuel tank (honestly I would have already when adding clutter because atmos desperately needs it anyway but I didn't think about it)
- Added a water vapor can
- Added vents to space for optionally draining loops

<img width="1279" height="1065" alt="image" src="https://github.com/user-attachments/assets/2b6f42f2-8e67-4ada-8b6b-b4ae43de365a" />

ArtisticRoomba reviewed (approved):
alright widgets you've had your fun

IProduceWidgets commented:
Probably for the best.
Its about time for the crimmas updates so I need to pivot to channeling my inner Mariah Carey
